### ISSUE TITLE
Allow different input rasters for features

### ISSUE THREAD

**Issue by raplima** [marcosbr/predictive-geology-maps#18]
**Is your feature request related to a problem? Please describe.**
Feedback from the first workshop. Users might want to explore different input features (rasters). Geochemical maps, for example, might be very helpul for identifying geology. As of now, the program is very limited on filenaming structure and features used for training and prediction, making it very cumbersome for users that want to try something different. 

**Describe the solution you'd like**
The program should be able to accept any number of input features (rasters), independent of filename.

**Comment by raplima** [comment]
I suggest modifying the "legacy"

```python
    def set_rasters_to_column(self):
        """Transform raster to numpy array to be used for training and eval
        """
```
and dropping the current
```python
    def prepare_to_fit(self):

```
in [PredMap](https://github.com/marcosbr/predictive-geology-maps/blob/main/predmap.py). 

The `set_rasters_to_column` is simpler and uses the input filenames. However, it does not save column names. It assembles the `self.X` attribute that is expected to be a numpy array, but that can easily be modified to be a pandas dataframe as the current code in `prepare_to_fit`. I also suggest declaring all class attributes in the `__init__` section and not create new ones in other methods (e.g, [here](https://github.com/marcosbr/predictive-geology-maps/blob/97f87734477f85da9f7a9acd90e03fb022f9bcc8/predmap.py#L66)).

**Comment by raplima** [comment]
Need also to think about "PCA column selection".

**Comment by marcosbr** [comment]
The issue #18 has been addressed in [commit](https://github.com/marcosbr/predictive-geology-maps/commit/295e654905491cc18c0c7266bf1d4254b52ea9b3) . The program now accepts any number of input rasters without the need for a specif file naming structure. The modifications also include the SIGLA_UNID being placed in the output directory #32. 

The legacy function    
```python
    def prepare_to_fit(self):
``` 
 no longer appears in the code, and its functions were incorporated in the 
```python
    def set_rasters_to_column(self):
```
I declared the class attributes in the __init__ section as suggested.
